client/README.md

```diff
@@ -191,4 +194,16 @@ options:
                         you identify the names of SQL objects in your source files, and 
                         specify target names for those objects in BigQuery. More info
                         please see https://cloud.google.com/bigquery/docs/output-name-mapping.
-```
\ No newline at end of file
+```
+
+## FAQ:
+### pip install couldn't find the right version for `google-cloud-storage`?
```

Comment:
I think for consistency you want back ticks also around "pip install"?

Reply:
Done.